v2/pkg/protocols/http/validate.go

```diff
@@ -0,0 +1,11 @@
+package http
+
+import "github.com/pkg/errors"
+
+func (request *Request) validate() error {
+	if request.Race && request.ReqCondition {
+		return errors.New("race and req-condition can't be used together")
```

Comment:
We should probably rather use the exact flag names, preferably between single or double quotes to avoid any kind of confusion

Reply:
Done